CMakeLists.txt

```diff
@@ -203,13 +205,15 @@ if(MSVC)
     MSVC/vasprintf-msvc.c 
     MSVC/arc4random.cc 
     MSVC/strptime.cc)
+elseif(ANDROID)
+    set(ALL_SRC_FILES ${ALL_SRC_FILES} 
+    LiteCore/Unix/strlcat.c
+    LiteCore/Unix/arc4random.cc
+    LiteCore/Android/unicode/ndk_icu.c)
 elseif(UNIX AND NOT APPLE)
-  set(ALL_SRC_FILES ${ALL_SRC_FILES} 
-    LiteCore/Android/strlcat.c
-    LiteCore/Android/arc4random.cc)
```

Comment:
Please make this change (LiteCore/Android/arc4random.cc -> LiteCore/Unix/arc4random.cc) in LiteCore/tests/CMakeLists.txt as well or CppTests will fail to compile

Reply:
Done.